### PR TITLE
Add option for Linkifying user name and channel name

### PR DESCRIPTION
**Pull request by tomoima525**
Hi! Thanks for the great service! I added a small feature to slack message step. 

# Done in this PR
- Added option to the step to linkify user name and channel name when posted from slack bot 

screen shot 2017-08-01 at 22 54 52

this is related issue #13

### CODE REVIEW

File: step.yml [hunk 1]
```diff
@@ -151,6 +151,16 @@ inputs:
         leave this option empty then the default one will be used.
       is_expand: true
       is_required: false
+  - link_names: "no"
```
Comment (viktorbenei) [inline step.yml]: Shouldn't we enable this by default? Seems useful enough to be default `yes` IMO

File: step.yml [hunk 2]
```diff
@@ -151,6 +151,16 @@ inputs:
...
+    opts:
+      title: "Link Names"
+      description: |
+        Link names such as @username
```
Comment (viktorbenei) [inline step.yml]: I'd also add a channel example here. E.g. `Linkify names in the message such as @slackbot or #random`

What do you think?

File: step.yml (hunk 1 above)
Comment (tomoima525) [inline step.yml]: Yeah, that is true! addressed  https://github.com/bitrise-io/steps-slack-message/pull/22/commits/bdff5e33551dc27fd8a4bd2a0fd649dbde4eb240

File: step.yml (hunk 2 above)
Comment (tomoima525) [inline step.yml]: Agree, should include channel name too -> https://github.com/bitrise-io/steps-slack-message/pull/22/commits/779e0023b34f5e9ffc66befe8f5a3883a2a63b3c

**Comment by tomoima525** [comment]
@viktorbenei thanks for the comments, I made fix to them

File: main.go [hunk 3]
```diff
@@ -31,6 +31,7 @@ type ConfigsModel struct {
 	EmojiOnError        string
 	IconURL             string
 	IconURLOnError      string
+	IsLinkNamesOn       bool
```
Comment (viktorbenei) [inline main.go]: one last minor thing, please rename this property to just `IsLinkNames`, to conform to the naming of the other properties

**Comment by viktorbenei** [comment]
Thank you @tomoima525 ! :) Found one last thing, sorry I missed this the first time.

**Comment by tomoima525** [comment]
@viktorbenei Sure thing! Just addressed

**Comment by viktorbenei** [comment]
Released in v2.6.1 - should be available in the steplib soon

**Pull request merged**